homeassistant/components/esphome/__init__.py

```diff
@@ -962,23 +978,3 @@ def name(self) -> str:
     def should_poll(self) -> bool:
         """Disable polling."""
         return False
-
-
-class EsphomeEntity(EsphomeBaseEntity[_InfoT, _StateT]):
```

Comment:
Got rid of this because `_on_state_update` was the only reason it existed in the first place (for `camera`). Now that we have it in the main EsphomeEntity class, I see no need for it anymore